packages/yew/src/suspense/suspension.rs

```diff
@@ -138,3 +125,24 @@ impl Drop for SuspensionHandle {
         self.inner.resume_by_ref();
     }
 }
+
+#[cfg_attr(documenting, doc(cfg(any(target_arch = "wasm32", feature = "tokio"))))]
+#[cfg(any(target_arch = "wasm32", feature = "tokio"))]
+mod feat_io {
+    use super::*;
+    use crate::io_coop::spawn_local;
+
+    impl Suspension {
+        /// Creates a Suspension that resumes when the [`Future`] resolves.
+        pub fn from_future(f: impl Future<Output = ()> + 'static) -> Self {
+            let (self_, handle) = Self::new();
+
+            spawn_local(async move {
+                f.await;
+                handle.resume();
+            });
```

Comment:
Instead of making this function available only outside of SSR environment, why not use `tokio::spawn`/`tokio::spawn_local` when we're on a WASM target?

Reply:
> I don't think Tokio works under wasm32 targets. You need wasm_bindgen_futures::spawn_local.

Tokio _could_ work but entirety of Yew would need to use it. Also certain features of tokio that relies on low level APIs won't work

Reply:
I think even if it could work, then we need to carry weight of tokio runtime with Yew.

If we choose to do so, what kind of benefit would it provide over `wasm_bindgen_futures` if anything that tries to use `fs`, `io` and `sync`(, etc...) cannot work and you need to spawn runtime with `wasm_bindgen_futures::spawn_local`?

Reply:
Makes sense. I must admit, my original comment was because I didn't see `tokio` there...